.github/workflows/cd.yml

```
@@ -0,0 +1,86 @@
+        with:
+          host: ${{ secrets.SSH_HOST }}
+          username: ${{ secrets.SSH_USERNAME }}
+          key: ${{ secrets.SSH_KEY }}
+          port: 22
+          source: "docker-compose.yml"
+          target: "/home/ubuntu/troublog/"
+
+      - name: Copy .env to EC2
+        uses: appleboy/scp-action@master
+        with:
+           host: ${{ secrets.SSH_HOST }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 11

(indentation)